android/source/VideoLocker/src/org/edx/mobile/view/CourseChapterListFragment.java

```diff
@@ -203,6 +204,10 @@ public void onStart() {
     public void onStop() {
         super.onStop();
         isActivityStarted = false;
+        //We need to cancel the getLastAccessed task if the fragment is stopped
+        if(getLastAccessedTask!=null){
+            getLastAccessedTask.cancel(true);
```

Comment:
Shouldn't there be a more general mechanism for canceling all related background tasks that no longer apply once the activity is stopped?  In this case, only the LastAccessedTask is canceled.  In the future, there may be other tasks that should also be cancelled.  Also, I would think other activities/pages would have similar requirements.